.hashibot.hcl

```
@@ -35,6 +35,30 @@ behavior "deprecated_import_commenter" "hashicorp_terraform" {
   EOF
 }
 
+behavior "deprecated_import_commenter" "sdkv1" {
+  import_regexp = "github.com/hashicorp/terraform-plugin-sdk/helper/(schema|resource|acctest)"
```

Let's go ahead and capture all the import paths that can be migrated:

```suggestion
  import_regexp = "github.com/hashicorp/terraform-plugin-sdk/(helper/(acctest|customdiff|logging|resource|schema|structure|validation)|terraform)"
```